controllers/articles.controller.js

```diff
@@ -30,4 +31,28 @@ exports.getArticleById = (req, res, next) => {
             console.error(err)
             res.status(500).send({ msg: 'Internal Server Error' });
         });
+};
+
+
+exports.getCommentsByArticle = (req, res, next) => {
+    const { article_id } = req.params
+    const regex = /^\d+$/;
+
+    if (!regex.test(article_id)) {
```

Comment:
this is not necessary because sql will do this check for you as article_id is a foreign key on comments and must be an integer